shared/AppInsightsCore/src/JavaScriptSDK/PerfManager.ts

```diff
@@ -148,19 +148,19 @@ export class PerfManager implements IPerfManager  {
 
         dynamicProto(PerfManager, this, (_self) => {
 
-            _self.create = (src: string, payloadDetails?: () => any, isAsync?: boolean): IPerfEvent => {
-                // TODO (newylie): at some point we will want to add additional configuration to "select" which events to instrument
+            _self.create = (src: string, payloadDetails?: () => any, isAsync?: boolean): IPerfEvent|null|undefined => {
+                // TODO (@MSNev): at some point we will want to add additional configuration to "select" which events to instrument
                 // for now this is just a simple do everything.
                 return new PerfEvent(src, payloadDetails, isAsync);
```

Comment:
When would this be null or undefined?

Reply:
As part of the documentation people can provide there own IPerfManager which can selectively opt-out of creating a IPerfEvent to indicate that it doesn't want to monitor the instrumented chunk of code.

OTEL are already creating their own version of IPerfManager and found this signature issue.

Reply:
But yes, for the default implementation it won't ever be, but for consistency with the *.d.ts, defining the return type to avoid issues in some environments and issues like #1324 which is just a signature issue